### PR TITLE
SAAS-830 Bulk Invite

### PR DESCRIPTION
**Pull request by peterSirotnak**
New pull request for SAAS-830 bulk invite, changes are the same.

### CODE REVIEW

**Review by TomasBlazek3PG — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts [hunk 1]
```diff
@@ -21,17 +21,63 @@ export const getStyles = (theme: GrafanaTheme) => {
       // style for Grafana's select
       // TODO: remove this after a select component is added to core-ui
       > div {
+        width: 97%;
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:24]: Why 97%?

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts [hunk 2]
```diff
@@ -21,17 +21,63 @@ export const getStyles = (theme: GrafanaTheme) => {
...
         padding-top: 7px;
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:25]: COuld this be done with spacing?

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts [hunk 3]
```diff
@@ -21,17 +21,63 @@ export const getStyles = (theme: GrafanaTheme) => {
...
+    iconButton: css`
+      align-self: flex-end;
+    `,
+    footerContainer: css`
+      display: flex;
+      justify-content: space-between;
+    `,
+    footerMessage: css`
+      vertical-align: bottom;
+      display: table-cell;
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:77]: Why table-cell?

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts [hunk 4]
```diff
@@ -21,17 +21,63 @@ export const getStyles = (theme: GrafanaTheme) => {
...
+      margin-top: 40px;
+      font-weight: bold;
+      color: gray;
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:80]: Is there suitable color in our scheme?

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts (hunk 3 above)
Comment (peterSirotnak) [reply src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts]: Forgoten styling, fixed

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts (hunk 1 above)
Comment (peterSirotnak) [reply src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts]: Removed added separate element for button that fixes this.

File: src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts (hunk 2 above)
Comment (peterSirotnak) [reply src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:24]: No 7px spacing present in the styles.

Comment (TomasBlazek3PG) [reply src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:24]: Not even close? Or needs to be pixel perfect? I didnt see how it looks so I am not sure

Comment (peterSirotnak) [reply src/pages/ManageOrganization/InviteMember/InviteMember.styles.ts:24]: it needs to be pixel perfect, for both fields to match.

**Review by TomasBlazek3PG — APPROVED** [review]

**Pull request merged**